modules/ext.PropertySuggester.EntitySelector.js

```diff
@@ -140,12 +140,12 @@
 		 * @return {string|null}
 		 */
 		_getPropertyContext: function() {
-			if ( this._isInNewStatementView() ) {
-				return 'item';
-			} else if ( this._isQualifier() ) {
+			if ( this._isQualifier() ) {
 				return 'qualifier';
 			} else if ( this._isReference() ) {
 				return 'reference';
+			} else if ( this._isInNewStatementView() ) {
+				return 'item';
```

Comment:
I don't see how guards might be useful here. Maybe a `switch ( null )` would be more appropriate here.